static/docs/commands-reference/root.md

```diff
@@ -2,12 +2,19 @@
 
 Returns relative path to project's directory.
 
-Can be used to build a path to a dependency, command, or output.
+## Synopsis
 
 ```usage
 usage: dvc root [-h] [-q] [-v]
 ```
 
+## Description
+
+While in project's sub-directory, sometimes developers may want to refer some
+file belonging to another directory. This command returns relative path to the
+DVC project's root directory from the current working directory. So, this
+command can be used to build a path to a dependency file, command, or output.  
```

Comment:
So, am I correct that this command was not built to be used inside DVC-files? (cc @efiop). The primary use case is to simplify developers' life when they run `dvc run`. And make `dvc run` executable effectively from any subdirectory. So, primarily it can be useful if I use a script that generates some stage files?

Reply:
@shcheklein Correct. But it can also be used in dvc files as well, just by using it in the command itself, so that it is evaluated each time the command is ran.